### PR TITLE
Update scripts in monorepo and subprojects

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard19**
## Summary

- Updated scripts
- Added shortcut scripts
- Updated docs
- Updated linter

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Import(end) PR

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -73,11 +73,11 @@ jobs:
 
       - name: Export texts (develop)
         if: ${{ env.BRANCH_NAME != 'master' }}
-        run: yarn workspace @corona-dashboard/cms lokalize:export --clean-json
+        run: yarn workspace @corona-dashboard/cms lokalize:import --clean-json
```
Comment (VWSCoronaDashboard24) [inline .github/workflows/main.yml:76]: Question: should we still use `lokalize` as it is a bit of legacy now we have Sanity in place?

File: packages/cms/src/data/data-structure.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ export const dataStructure = {
       "birthyear_range",
       "age_group_total",
     ],
-    booster_coverage: ["percentage"],
+    booster_coverage: ["age_group", "percentage"],
```
Comment (VWSCoronaDashboard24) [inline packages/cms/src/data/data-structure.ts:77]: Just checking; is this change intentional?

**Review by VWSCoronaDashboard24 — APPROVED** [review]
